packages/@aws-cdk/aws-stepfunctions/lib/condition.ts

```diff
@@ -2,13 +2,110 @@
  * A Condition for use in a Choice state branch
  */
 export abstract class Condition {
+
+  /**
+   * Tests if variable is present
+   */
+  public static isPresent(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsPresent, true);
+  }
+
+  /**
+   * Tests if variable is not present
+   */
+  public static isNotPresent(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsPresent, false);
+  }
+
+  /**
+   * Tests if variable is a string
+   */
+  public static isString(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsString, true);
+  }
+
+  /**
+   * Tests if variable is not a string
+   */
+  public static isNotString(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsString, false);
+  }
+
+  /**
+   * Tests if variable is numeric
+   */
+  public static isNumeric(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsNumeric, true);
+  }
+
+  /**
+   * Tests if variable is not numeric
+   */
+  public static isNotNumeric(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsNumeric, false);
+  }
+
+  /**
+   * Tests if variable is boolean
+   */
+  public static isBoolean(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsBoolean, true);
+  }
+
+  /**
+   * Tests if variable is not boolean
+   */
+  public static isNotBoolean(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsBoolean, false);
+  }
+
+  /**
+   * Tests if variable is a timestamp
+   */
+  public static isTimestamp(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsTimestamp, true);
+  }
+
+  /**
+   * Tests if variable is not a timestamp
+   */
+  public static isNotTimestamp(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsTimestamp, false);
+  }
+
+  /**
+   * Matches if variable is not null
+   */
+  public static isNotNull(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsNull, false);
+  }
+  /**
+   * Matches if variable is Null
+   */
+  public static isNull(variable: string): Condition {
+    return new VariableComparison(variable, ComparisonOperator.IsNull, true);
+  }
   /**
    * Matches if a boolean field has the given value
    */
   public static booleanEquals(variable: string, value: boolean): Condition {
     return new VariableComparison(variable, ComparisonOperator.BooleanEquals, value);
   }
 
+  /**
+   * Matches if a boolean field has the given value
+   */
+  public static booleanEqualsPath(variable: string, value: string): Condition {
```

Comment:
wondering if `Path` is the right suffix for these methods?
I think we've been explicit about adding `fromJsonPath` when naming these methods. what do you think?

Reply:
Yeah - you have a point. In the Dynamo attribute value context json path is used.

I guess in this case I took inspiration from the cloudformation naming which doesn't use the json path. 

I'm happy to stick with json path... don't have strong feelings against it